sentry/templates/deployment-sentry-worker.yaml

```diff
@@ -77,6 +92,10 @@ spec:
         - name: C_FORCE_ROOT
           value: "true"
 {{ include "sentry.env" . | indent 8 }}
+        {{ if .Values.sentry.worker.customCA }}
+        - name: REQUESTS_CA_BUNDLE
+          value: /etc/pki/ca-trust/custom/{{ default "ca.crt" .Values.sentry.worker.customCA.item }}
+        {{ end }}
```

Comment:
```suggestion
```
This is not needed because we will package all the CA into a single file, so the REQUEST library will be able to read the new CustomCA from there.

Same for the sentry-web deployment, we should remove this variable from there.